LICENSE

```diff
@@ -1,7 +1,7 @@
-Copyright 2017 Alex Jover Morales, @alexjoverm <alexjovermorales@gmail.com>
+Copyright 2017 @--username-- <--usermail-->
```

Comment:
I think here the @ symbol should be removed, since git config name takes a name, not a twitter mention